repos/fdbt-site/src/pages/createCaps.tsx

```diff
@@ -1,34 +1,55 @@
 import React, { ReactElement, useState } from 'react';
-import CapTable from '../components/CapTable';
+import { ExpiryUnit } from '../interfaces/matchingJsonTypes';
 import CsrfForm from '../components/CsrfForm';
 import ErrorSummary from '../components/ErrorSummary';
-import FormElementWrapper from '../components/FormElementWrapper';
-import { CAPS_ATTRIBUTE } from '../constants/attributes';
-import { Cap, ErrorInfo, NextPageContextWithSession } from '../interfaces';
+import FormElementWrapper, { FormGroupWrapper } from '../components/FormElementWrapper';
+import { CREATE_CAPS_ATTRIBUTE } from '../constants/attributes';
+import { CapInfo, ErrorInfo, NextPageContextWithSession } from '../interfaces';
 import { isWithErrors } from '../interfaces/typeGuards';
 import { FullColumnLayout } from '../layout/Layout';
-import { getCsrfToken } from '../utils';
+import { getAndValidateNoc, getCsrfToken } from '../utils';
 import { getSessionAttribute } from '../utils/sessions';
+import { upperFirst } from 'lodash';
+import { getCapByNocAndId } from '../data/auroradb';
 
 const title = 'Create Caps - Create Fares Data Service';
 const description = 'Create caps page of the Create Fares Data Service';
 
+const daysOfWeek = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
```

Comment:
this is defined in 4 different places in the repo - can we remove all of them and create a DAYS_OF_WEEK variable in an appropriate place? e.g. repos/fdbt-site/src/constants/index.ts